waspc/packages/deploy/src/providers/railway/index.ts

```diff
@@ -111,6 +111,10 @@ function makeRailwaySetupCommand(): Command {
       "--workspace [workspace]",
       "the Railway workspace to use if a new project needs to be created (if not provided, will ask interactively)",
     )
+    .option(
+      "--db-image [dbImage]",
+      "custom Docker image for the Postgres database (e.g., postgis/postgis for PostGIS extensions)",
```

Comment:
Where will it pull this image from? DockerHub? Is that why org/image is enough? If so, should we document that? If not, does it take more info then?

Reply:
We're not doing anything special, just passing the image name to the Docker CLI. By convention, images that don't specify a registry go to `docker.io`, and for all others, the registry domain is _part of the image name_; so basically none of the documentation dwells on this, and the behavior is expected by users and the ecosystem.